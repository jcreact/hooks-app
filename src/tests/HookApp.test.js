import React from 'react';
import { shallow } from 'enzyme';

import { HookApp } from '../HookApp.jsx';

describe('Pruebas en "HookApp.jsx".', () => {

    const component = shallow(<HookApp />);

    test('Debe mostrar HookApp correctamente', () => {

        expect(component).toMatchSnapshot();
    });

    test(`Debe contener un h1 con 'Hooks App'.`, () => {
        expect(component.find('h1').exists()).toBeTruthy();
        expect(component.find('h1').text()).toBe('Hooks App');
    });
});
