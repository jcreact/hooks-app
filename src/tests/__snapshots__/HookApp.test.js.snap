// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en "HookApp.jsx". Debe mostrar HookApp correctamente 1`] = `
<Fragment>
  <h1>
    Hooks App
  </h1>
</Fragment>
`;
